rm: CI LUKS encryption (#768)
foundriesio/docs: source/reference-manual/linux/linux-disk-encryption.rst

key used for data encryption is protected and unlocked using a
passphrase.

> Users can choose their own passphrase; if they don't, LmP will generate
> one for them.

Reviewer: ```suggestion
You can choose your own passphrase; if you do not, LmP will generate one for you.
```
